ui/src/components/ParcelSize.jsx

```diff
@@ -6,16 +6,24 @@ import "../styles/SendingInfo.css";
 import AttributesSelection from "./AttributesSelection";
 
 function ParcelSize(props) {
+  var additionalPrice = 0;
+  additionalPrice = props.selectedAttributes.map((attribute) => {
+    return additionalPrice + attribute.additionalPrice;
+  });
+  if (additionalPrice.length !== 0) additionalPrice = parseInt(additionalPrice);
+  else additionalPrice = 0;
```

Comment:
This won't work if we have more than one attribute. Instead of `map()` use `reduce()` or `forEach()` methods. Also move price calculation to separate file. For example `src/services/priceCalculation.js`.